src/huggingface_hub/commands/user.py

```diff
@@ -157,8 +157,9 @@ def run(self):
             print("Not logged in")
             exit()
         try:
-            user, orgs = self._api.whoami(token)
-            print(user)
+            info = self._api.whoami(token)
+            print(info["name"])
```

Comment:
```suggestion
```

Reply:
Why? The `print` is wanted here, this is the CLI.